app/com/linkedin/drelephant/analysis/AnalyticJob.java

```diff
@@ -267,6 +272,9 @@ public AppResult getAnalysis() throws Exception {
     result.totalDelay = hadoopAggregatedData.getTotalDelay();
     result.resourceWasted = hadoopAggregatedData.getResourceWasted();
 
+    String pigParent = data.getConf().getProperty("pig.parent.jobid");
```

Comment:
We may not want to call it pigParent since we can have this feature for other job types as well. Also this should be pluggable.